playground/terraform/infrastructure/cloudbuild-manual-setup/README.md

```diff
@@ -0,0 +1,154 @@
+    - Security Admin
+    - Service Account User
+- [gcloud CLI](https://cloud.google.com/sdk/docs/install-sdk)
+- An existing GCP Bucket to save Terraform state - `state_bucket`
+- An existing GCP Bucket to save Cloud Build logs - `logs_bucket`
+- DNS name for your Playground deployment instance
+- [Terraform](https://www.terraform.io/)
+- [Apache Beam GitHub](https://github.com/apache/beam) repository cloned locally
+
+## 1. Setup the Google Cloud Build for your GCP project
+
+The `playground/terraform/infrastructure/cloudbuild-manual-setup/01.setup` provisions dependencies required to setup Cloud Build for Playground:
+- Required API services
+- Cloud Build service account - `playground-cloudbuild-sa`
+- IAM permissions for Cloud Build service account - `playground-cloudbuild-sa`
+
+#### To execute the module:
+
+1. Create a folder `playground/terraform/environment/{environment_name}` (further referred to as `environment_name`) to define a new environment for Playground.
+2. Create `terraform.tfvars` file in the newly created `environment_name` folder.
+3. Fill the `terraform.tfvars` configuration file as follows:
+
+
+```console
+project_id                    = "project_id"                # Your Project ID
+region                        = "us-central1"               # Your GCP region name where resources will be provisioned
+location                      = "us-central1-b"             # Select the deployment location from available in the specified region
+cloudbuild_service_account_id = "playground-cloudbuild-sa"  # The name of Cloud Build service account
+github_repository_name        = "beam"                      # The name of the GitHub repo to be connected with Cloud Build. Example: In https://github.com/example/foo is 'foo'
+github_repository_owner       = "repo_owner_name"           # Owner of the GitHub repo to be connected with Cloud Build. Example: In https://github.com/example/foo is 'example'.
+github_repository_branch      = "branch_name"               # The name of the GitHub repo branch to be connected with Cloud Build
+
+network_name                  = "network_name"              # GCP VPC Network Name for Playground deployment
+gke_name                      = "playground-backend"        # Playground GKE Cluster name
+state_bucket                  = "bucket_name"               # GCS bucket name for Beam Playground temp files
+```
+
+4. Create and fill `state.tfbackend` configuration file in same `environment_name` folder as follows:
+
+
+```
+bucket = "bucket_name"     # The name of bucket - will be used for terraform tfstate file
+```
+
+5. Push the `environment_name` folder to your GitHub repository `github_repository_name` and branch `github_repository_branch`.
+(Cloud Build needs file be present in GitHub repository it is connected to)
+
+**Note:** Some regions can be prohibited for Cloud Build. Please see [Cloud Build locations](https://cloud.google.com/build/docs/locations) for supported locations.
```

Comment:
@ruslan-ikhsan can you rephrase more friendly? Something like:
**Note:** Please see [Cloud Build locations](https://cloud.google.com/build/docs/locations) for the list of all supported locations.